qa/suites/upgrade/luminous-x/parallel/5-final-workload/rgw_ragweed_check.yaml

```diff
@@ -0,0 +1,11 @@
+meta:
+- desc: |
+    ragweed check for rgw
+rgw-final-workload:
+  full_sequential:
+  - ragweed:
+      client.1:
+        force-branch: ceph-master
```

Comment:
if we cherry-pick this into mimic we'll need to change this line

Reply:
i'll go ahead and create these branches